hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -304,6 +308,18 @@ public class HoodieTableConfig extends HoodieConfig {
       .sinceVersion("1.0.0")
       .withDocumentation("When set to true, the table can support reading and writing multiple base file formats.");
 
+  public static final ConfigProperty<PartialUpdateMode> PARIL_UPDATE_MODE = ConfigProperty
+      .key("hoodie.write.partial.update.mode")
+      .defaultValue(PartialUpdateMode.KEEP_VALUES)
```

Comment:
do we even need a default value for this config? 
if partial update is enabled the table property will have to set w/ the right value anyways. So, why bother setting default values.